### PR TITLE
InvalidResourceException file name

### PR DESCRIPTION
**Pull request by DancZer**
Added filen ame to the InvalidResourceException,

### CODE REVIEW

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ private function parseFile($file)
         $source = str_replace('http://www.w3.org/2001/xml.xsd', $location, $source);
 
         if (!@$dom->schemaValidateSource($source)) {
-            throw new InvalidResourceException(implode("\n", $this->getXmlErrors($internalErrors)));
+            throw new InvalidResourceException("File:$file\n".implode("\n", $this->getXmlErrors($internalErrors)));
```
Comment (jakzal) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: Let's be more verbose and say something like *The "file.xlf" is not a valid resource*.

Also, our coding standard favours single quotes, and sprintf should be used to create an exception message:

```php
throw new InvalidResourceException(sprintf('The "%s" is not a valid resource: ', $file, implode("\n", $this->getXmlErrors($internalErrors))));
```

File: src/Symfony/Component/Translation/Loader/XliffFileLoader.php [hunk 2]
```diff
@@ -148,7 +148,7 @@ private function parseFile($file)
         $source = str_replace('http://www.w3.org/2001/xml.xsd', $location, $source);
 
         if (!@$dom->schemaValidateSource($source)) {
-            throw new InvalidResourceException(implode("\n", $this->getXmlErrors($internalErrors)));
+            throw new InvalidResourceException(sprintf('The "%s" is not a valid resource: ', $file, implode("\n", $this->getXmlErrors($internalErrors))));
```
Comment (iltar) [inline src/Symfony/Component/Translation/Loader/XliffFileLoader.php]: `The "my_file.xml" is not a valid resource`, This sounds a bit weird. What if you format it like this? 

`'Invalid resource provided: "%s"; Errors: %s'`, note that you forgot the second `%s` in your commit.

**Comment by Tobion** [comment]
:+1: should be merged in 2.3

**Comment by fabpot** [comment]
Thank you @DancZer.

**Pull request closed without merging**